core/lib/refinery/plugin.rb

```diff
@@ -1,5 +1,16 @@
 module Refinery
   class Plugin
+    META = {
```

Comment:
I like simple.  I think listing them out in this hash is simple.  But I'm okay removing it from there if that's really what you'd like.  I totally agree with letting the user ("developer user", not the "logged in user") organize plugins.  Using a range of 1-50 helps the developer accomplish this simply.  It is a replacement for the `plugin_priority` setting.  I'd prefer to get that next PR in before we make a decision on the 1-50 if you don't mind.